### ISSUE TITLE
[Meds / Improvements] Make accessibility enhancements to the Medications list view

### ISSUE THREAD

**Issue by michaelbrodsky-a6** [department-of-veterans-affairs/va.gov-team#118720]
## Context 
This ticket emerged from a review of the previous Medications team's Jira board, and is captured in row 65 of this [Jira Board Inventory](https://dvagov.sharepoint.com/:x:/r/sites/HealthApartment/Shared%20Documents/Medications/Internal%20Planning/Open%20Tickets%20-%20Jira%20Board%20Inventory%20-%20Medications%20-%20MHV%20on%20VA.gov%20-%20August%202025.xlsx?d=w17103d36bdf64699838ef89fe1a965af&csf=1&web=1&e=MOXAYZ) spreadsheet.

## Background
Currently, when a Vet applies or clears a filter, the page content updates visually bit there isn't any clear action-based confirmation for screen reader folks. This should be changed. We also need to clearly announce the action that caused the change. 

## Description
Make accessibility enhancements to the Medications list view.

## Details
Note: There's additional detail in [this Jira ticket](jira.devops.va.gov/browse/MHV-71097).

## Resources
- Here's a link to the [original Jira ticket](jira.devops.va.gov/browse/MHV-71097) with additional details.

## Notes
-

## Tasks
- Gain access to Jira
- Make accessibility enhancements to the Medications list view
- QA review

## Acceptance Criteria
- [ ] Accessibility enhancements to the Medications list view
- [ ] a11y review
- [ ] QA review
